importer/templates/importer/upload.html

```diff
@@ -0,0 +1,6 @@
+<h1>upload here</h1>
```

Comment:
templates should always extend base.html. https://code.djangoproject.com/wiki/DosAndDontsForApplicationWriters